src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -21,7 +21,14 @@ namespace Microsoft.AspNet.Mvc.TagHelpers
     /// <remarks>
     /// <see cref="FallbackSrc" /> and <see cref="FallbackTestExpression" /> are required to not be
     /// <c>null</c> or empty to process.
```

Comment:
nit: please correct this sentence (or just remove it since "it's complicated" might be the best description) as well.  check w/ @DamianEdwards .